card/mod/05_standard/set/type/user.rb

```diff
@@ -37,8 +38,8 @@
   end
 end
 
-
-event :setup_as_bot, before: :check_permissions, on: :create, when: proc{ |c| Card::Env.params[:setup] } do
+event :setup_as_bot, before: :check_permissions, on: :create,
+      when: proc { |c| Card::Env.params[:setup] } do
```

Comment:
Unused block argument - `c`. You can omit the argument if you don't care about it.<br>Align the elements of a hash literal if they span more than one line.